MemorialHouse/MHPresentation/MHPresentation/Source/EditBook/View/EditPageCell.swift

```diff
@@ -39,31 +47,267 @@ final class EditPageCell: UITableViewCell {
+                case let .error(message):
+                    MHLogger.error(message) // 더 좋은 처리가 필요함
+                }
+            }.store(in: &cancellables)
+    }
     
-    // MARK: - TouchEvent
-    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
-        textView.becomeFirstResponder()
+    // MARK: - Method
+    func configure(viewModel: EditPageViewModel) {
+        self.viewModel = viewModel
+        configureBinding()
+        input.send(.pageWillAppear)
+    }
+    
+    // MARK: - Helper
+    private func configurePage(page: Page) {
+        let mergedText = mergeStorageInformation(
+            text: page.text,
+            attachmentMetaData: page.metadata
+        )
+        textStorage?.setAttributedString(mergedText)
+    }
+    private func mediaAddedWithData(media: MediaDescription, data: Data) {
+        let attachment = MediaAttachment(
+            view: MHPolaroidPhotoView(), // TODO: - 수정 필요
+            description: media
+        )
+        attachment.configure(with: data)
+        let text = NSMutableAttributedString(attachment: attachment)
+        text.addAttributes(defaultAttributes,
+                           range: NSRange(location: 0, length: 1))
+        textStorage?.append(text)
+    }
+    private func mediaAddedWithURL(media: MediaDescription, url: URL) {
+        let attachment = MediaAttachment(
+            view: MHPolaroidPhotoView(),// TODO: - 수정 필요
+            description: media
+        )
+        attachment.configure(with: url)
+        let text = NSMutableAttributedString(attachment: attachment)
+        text.addAttributes(defaultAttributes,
+                           range: NSRange(location: 0, length: 1))
+        textStorage?.append(text)
+    }
+    private func mediaLoadedWithData(media: MediaDescription, data: Data) {
+        let attachment = findAttachment(by: media)
+        attachment?.configure(with: data)
+    }
+    private func mediaLoadedWithURL(media: MediaDescription, url: URL) {
+        let attachment = findAttachment(by: media)
+        attachment?.configure(with: url)
+    }
+    private func saveContents() {
+        guard let textStorage else { return }
         
-        super.touchesBegan(touches, with: event)
+        input.send(.didEditPage(attributedText: textStorage))
+    }
+    /// Text에서 특정 Attachment를 찾아서 적용합니다.
+    private func findAttachment(
+        by media: MediaDescription
+    ) -> MediaAttachment? {
+        var attachment: MediaAttachment?
+        guard let textStorage else { return attachment }
+        textStorage
+            .enumerateAttribute(
+                .attachment,
+                in: NSRange(location: 0, length: textStorage.length)
+            ) { value, _, _ in
+                if let mediaAttachment = value as? MediaAttachment,
+                   mediaAttachment.mediaDescription.id == media.id {
+                    attachment = mediaAttachment
+                    return
+                }
```

Comment:
P3:
```suggestion
                guard let mediaAttachment = value as? MediaAttachment,
                      mediaAttachment.mediaDescription.id == media.id else { return }
                attachment = mediaAttachment
```

정말 별것 아니긴한데.. guard문을 쓰면 depth가 덜 들어가서 가독성이 높아지는 느낌..? 입니당 (굉장히 개인적인 의견..)